packages/app/package.json

```diff
@@ -30,22 +30,22 @@
     "dist:prod": "npm run build && electron-builder --publish never",
     "dist:prod:all": "npm run dist:prod --",
     "TEST COMMANDS": "=========================================================",
-    "test": "cross-env TZ=UTC jest --config=jest-unit.config.js",
-    "test:all": "cross-env TZ=UTC jest --config=jest.config.js",
-    "test:coverage": "npm run test -- --coverage",
+    "test": "cross-env TZ=UTC jest --config=jest.config.js",
+    "test:coverage": "npm run test",
```

Comment:
Do we need this kind of scripts that are just invoking other scripts?

Reply:
fixed https://github.com/mudita/mudita-center/pull/1550/commits/86159d6af862743e3291cea7d2f9a8f4d891b2b0#diff-9fa008865bef344ab2f931a4ddf20163882b86ee764718adacf098dc19991946